cmd/skupper/skupper.go

```diff
@@ -531,7 +542,7 @@ func NewCmdUnexpose(skupperCli SkupperServiceClient) *cobra.Command {
 		RunE:   skupperCli.Unexpose,
 	}
 	cmd.Flags().StringVar(&unexposeAddress, "address", "", "Skupper address the target was exposed as")
-	skupperCli.UnexposeFlags(cmd)
+	cmd.Flags().StringVar(&unexposeNamespace, "target-namespace", "", "Target namespace for exposed service")
```

Comment:
This `target-namespace` should also be moved to Kube's implementation of `UnexposeFlags`.

Reply:
If I didn't miss anything, all namespace-related flags were moved to the respectives k8s implementations